src/components/programs/DeleteProgram.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+	HStack,
+	Icon,
+	Text,
+	useBreakpointValue,
+	useDisclosure,
+	useColorMode,
+	useColorModeValue,
+} from '@chakra-ui/react';
+import { useState } from 'react';
+import { IoTrashSharp } from 'react-icons/io5';
+
+import { useDriveContext } from 'contexts/drive';
+import { useUserContext } from 'contexts/user';
+
+import Button from 'components/Button';
+import Modal from 'components/Modal';
+import type { IPCProgram } from 'types/types';
+import { textColorMode } from 'config/colorMode';
+
+type DeleteProgramProps = {
+	program: IPCProgram;
+};
+
+const DeleteProgram = ({ program }: DeleteProgramProps): JSX.Element => {
+	const { user } = useUserContext();
+	const { programs, setPrograms } = useDriveContext();
+
+	const [isLoading, setIsLoading] = useState(false);
+	const { isOpen, onOpen, onClose } = useDisclosure();
+
+	const isDrawer = useBreakpointValue({ base: true, sm: false }) || false;
+	const textColor = useColorModeValue(textColorMode.light, textColorMode.dark);
+	const { colorMode } = useColorMode();
+
+    const DeleteActualProgram = async () => {
```

Comment:
```suggestion
    const deleteActualProgram = async () => {
```
Use PascalCase only for component names, not for functions :wink: